src/components/modals/about/AboutHubModal.tsx

```diff
@@ -20,7 +21,10 @@ export default function AboutHubModal({
 }: AboutHubModalProps) {
   const router = useRouter()
   const { data: hub } = getSpaceBySpaceIdQuery.useQuery(hubId)
-  const [isOpenMetadataModal, setIsOpenMetadataModal] = useState(false)
+
+  const [openedModalType, setOpenedModalType] = useState<
+    '' | 'metadata' | 'qr'
+  >('')
```

Comment:
Extract the type, because you use it in 2 places

+ also use null instead ""

Reply:
Updated to use null

but for the type extraction, I think its not good, because the hub modal and chat modal may have different modals
for example, in my latest commit, I add the `leave confirmation` modal state to use the same `openedModalType` state.